sources/academy/tutorials/node_js/dealing_with_dynamic_pages.md

```diff
@@ -144,7 +144,7 @@ So, we've gotta scroll down the page to load these images. Luckily, because we'r
 
 ```js
 import { PuppeteerCrawler, utils, Dataset } from 'crawlee';
-import cheerio from 'cheerio';
+import * as cheerio from 'cheerio';
```

Comment:
Can't we just get rid of the cheerio import here whatsoever? It's unused(?), and might confuse beginners that to use the `parseWithCheerio` helper, you need to import the cheerio library.

Reply:
right, saw it in one place but forgot to check others, good catch